src/classes/GE_PaymentServices.cls

```diff
@@ -122,13 +122,23 @@ public with sharing class GE_PaymentServices {
         return config.get(PS_IntegrationServiceConfig.BASE_URL);
     }
 
-    public String getJwt() {
-        JWTPayload jwtPayload = new JWTPayload(config.get(PS_IntegrationServiceConfig.PRODUCTID));
+    public String getJwtForPurchase() {
+        JWTPayload jwtPayload = new JWTPayload(config.get(PS_IntegrationServiceConfig.PRODUCTID),
+                PS_IntegrationServiceConfig.PURCHASE_ROLE);
         String jwt = UTIL_Jwt.getSignedJWTFromString(
                 jwtPayload.getAsString(),
                 config.get(PS_IntegrationServiceConfig.API_KEY)
         );
+        return jwt;
+    }
 
+    public String getJwtForToken() {
+        JWTPayload jwtPayload = new JWTPayload(config.get(PS_IntegrationServiceConfig.PRODUCTID),
+                PS_IntegrationServiceConfig.TOKEN_ROLE);
```

Comment:
```suggestion
            PS_IntegrationServiceConfig.TOKEN_ROLE);
```

Reply:
this code has been removed after merging with @RadSFDeveloper's PR.